pytest_django/plugin.py

```diff
@@ -89,6 +89,14 @@
              'assertNotContains', 'assertNumQueries', 'assertQuerysetEqual',
              'assertRaisesMessage', 'assertRedirects', 'assertTemplateNotUsed',
              'assertTemplateUsed', 'assertXMLEqual', 'assertXMLNotEqual'),
+    # from https://github.com/django/django/blob/stable/1.10.x/django/test/testcases.py#L155
+    #  and https://github.com/django/django/blob/stable/1.10.x/django/test/testcases.py#L797
+    (1, 10): ('assertContains', 'assertFieldOutput', 'assertFormError',
+              'assertFormsetError', 'assertHTMLEqual', 'assertHTMLNotEqual',
+              'assertInHTML', 'assertJSONEqual', 'assertJSONNotEqual',
+              'assertNotContains', 'assertNumQueries', 'assertQuerysetEqual',
+              'assertRaisesMessage', 'assertRedirects', 'assertTemplateNotUsed',
+              'assertTemplateUsed', 'assertXMLEqual', 'assertXMLNotEqual'),
 }
```

Comment:
While the duplication here is not exactly _accidental_, I think it's save to say that it's merely _coincidental_. There is most probably no deep underlying principle behind what functions are shared between several versions and which aren't.

Thus, extracting the common function names to eliminate duplication might make this configuration less clear and maintainable instead of more so. (cf. [Beware the Share](http://programmer.97things.oreilly.com/wiki/index.php)/Beware_the_Share) by Udi Dahan